imports/ui/case/case-messages.jsx

```diff
@@ -269,6 +269,7 @@ class CaseMessages extends Component {
         <div className='flex-grow relative'>
           <input type='text' placeholder='Type your response' ref='messageInput'
             onChange={this.handleMessageInput.bind(this)} value={message}
+            onKeyPress={event => event.key === 'Enter' && this.handleCreateMessage(event)}
```

Comment:
nit: use `if (condition) statement` instead of tricksy boolean operators.

Reply:
Unlike relying on pressing on the button, this could create empty content comments. You should check if 'message' is not empty